src/vision/calib/Calibration.cpp

```diff
@@ -0,0 +1,210 @@
+#include "MonoCalibrator.hpp"
+#include "wfov_camera_msgs/WFOVImage.h"
+#include <cv_bridge/cv_bridge.h>
+#include <sensor_msgs/image_encodings.h>
+#include <string>
+#include <vector>
+
+using namespace cv_bridge;
+
+Settings s;
+
+bool calibFinished = false;
+vector<vector<Point2f>> imagePoints;
+Mat cameraMatrix, distCoeffs;
+Size imageSize;
+int mode = CAPTURING;
```

Comment:
Where is mode defined? If it's a robosub-specific enum, it should probably be namespaced. E.g.
VisionProcessing::Mode::Capturing
This can be done by doing enum classes.